Lazaryshyn/Decorator design pattern/src/Decorator.java

```diff
@@ -0,0 +1,15 @@
+public class Decorator implements Car {
+
+        protected Car car;
+
+        public Decorator(Car c){
+            this.car=c;
```

Comment:
Please give longer names to variables next time

Reply:
okey